packages/tools/devtools/devtools-example/src/Container.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ModelContainerRuntimeFactory } from "@fluid-example/example-utils";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { IContainer } from "@fluidframework/container-definitions";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+
+import { CollaborativeDevtoolExampleAppObjects } from "./FluidObject";
+
+/**
+ * The CollaborativeDevtoolExampleAppModel is the model for our Fluid object.
+ * It contains the data we want to share with other clients.
+ */
+export interface ICollaborativeDevtoolExampleAppModel {
```

Comment:
Nit: We can probably just call this `AppModel`, since it's specific to the example app, and there is only 1 model.

Reply:
Renamed